ui/v2.5/src/models/list-filter/criteria/is-missing.ts

```diff
@@ -1,8 +1,8 @@
 import { CriterionModifier } from "src/core/generated-graphql";
 import { Criterion, CriterionType, ICriterionOption } from "./criterion";
 
-export class IsMissingCriterion extends Criterion {
-  public type: CriterionType = "isMissing";
+export class SceneIsMissingCriterion extends Criterion {
+  public type: CriterionType = "sceneIsMissing";
```

Comment:
In terms of abstracting, I did the following:

Keep `IsMissingCriterion` as an abstract class as follows:
```
export abstract class IsMissingCriterion extends Criterion {
  public parameterName: string = "is_missing";
  public modifierOptions = [];
  public modifier = CriterionModifier.Equals;
  public value: string = "";
}
```

Have the two concrete classes extend from `IsMissingCriterion` and remove the `parameterName`, `modifierOptions`, `modifier`, and `value` fields. 

I also changed the `is_missing` assignments in `filter.ts` to cast to `IsMissingCriterion` instead of the concrete classes.

I can push these changes myself if you prefer.

Functionally not much different, but I guess it saves some LOC.

Reply:
Yeah, good suggestions, I'm not going to be able to push these changes till tomorrow, so if you've got the time, go ahead 👍